src/sagemaker/environment_variables.py

```diff
@@ -19,6 +19,7 @@
 
 from sagemaker.jumpstart import utils as jumpstart_utils
 from sagemaker.jumpstart import artifacts
+from sagemaker.jumpstart.enums import EnvVariableUseCase
```

Comment:
Although we only use the env with hosting APIs (`CreateModel`), [CreateTrainingJob](https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_CreateTrainingJob.html#sagemaker-CreateTrainingJob-request-Environment) also accepts an `Environment`.

In future PRs, should we add a `scope` in `environment_variables.retrieve_default()`?

Reply:
yes, we can introduce scope later once there's an immediate need and we have more concrete details